### DISCUSSION TITLE
pulse analyzer's choice for when to produce a triq.org link

### DISCUSSION THREAD

**Discussion by compugician** [merbanan/rtl_433#2190] [Q&A]
In [pulse_analyzer.c](https://github.com/merbanan/rtl_433/blob/282051b39da79336f2680d8f234dbf0a776a041a/src/pulse_analyzer.c#L346) a triq.org link will be displayed only if (hist_timings.bins_count <= 8), I'm wondering why that is? 

I can work around that by outputting to file, obviously, but the link in stdout is extremely convenient for quick interrogation.

I'm thinking I'm probably missing something? is there a particular reason for the choice of <=8? or can it be a higher threshold?

**Comment by zuckschwerdt** [comment]
The format used supports only up to 8 different pulse sizes, it could be extented but is really only useful for PWM, PPM, MC -- formats with a limited set of pulse (and gap) lengths.

Perhaps look at https://github.com/merbanan/rtl_433/tree/feat-openinbrowser -- some things to work out there, but it should be a better integration in the long run.
